UIforETW/Utility.cpp

```diff
@@ -278,59 +482,64 @@ void SmartEnableWindow(HWND Win, BOOL Enable)
 
 std::wstring GetFilePart(const std::wstring& path)
 {
-	const wchar_t* pLastSlash = wcsrchr(path.c_str(), '\\');
-	if (pLastSlash)
-		return pLastSlash + 1;
+	UIETWASSERT(path.size() > 0);
+	const size_t lastSlash = path.find_last_of( L'\\' );

```

Comment:
Remember that L'\\' is the same as '\\', so that L is just decoration. This is true for all of the 7-bit ASCII characters.